lib/dockerregistry/inventory_test.go

```diff
@@ -104,7 +104,7 @@ func TestReadJSONStream(t *testing.T) {
 	for _, test := range tests {
 		var sr stream.Fake
 		sr.Bytes = []byte(test.input)
-		h, _ := sr.Produce()
+		h, _, _ := sr.Produce()
```

Comment:
hmm why aren't we handling these?

Reply:
It's because ReadJSONStream only reads/parses JSON from stdout, not stderr. Also, the stream.Fake only produces output in stdout, so these are always empty/nil.

Reply:
Oops I meant json.Consume()

Reply:
if the fake ever does produce errors we won't be caching them though.

Reply:
True. Will add.